ios/RNSScreen.mm

```diff
@@ -416,7 +416,7 @@ - (void)notifyHeaderHeightChange:(double)headerHeight
       [[RNSHeaderHeightChangeEvent alloc] initWithEventName:@"onHeaderHeightChange"
                                                    reactTag:[NSNumber numberWithInt:self.tag]
                                                headerHeight:headerHeight];
-  [[RCTBridge currentBridge].eventDispatcher sendEvent:event];
+  [[RCTBridge currentBridge].eventDispatcher notifyObserversOfEvent:event];
```

Comment:
yep, I fixed it in another library: [react-native-pager-view](https://github.com/callstack/react-native-pager-view/pull/809)
That's a terrible approach, we have to fix though.

ScrollView works because it does [this](https://github.com/facebook/react-native/blob/main/packages/react-native/Libraries/Components/ScrollView/ScrollView.js#L52-L54). Which basically import the whole paper renderer to register the `RCTEventEmitter`.

We will work on a better solution for 0.75, as the fix is more convoluted than just registering the module, unfortunately, and it is unlikely that we would be able to have the proper fix in time for 0.74 stable.

As a work around, you can apply the same fix of above.

Reply:
I just checked and the code I trigger here makes it work with `Animated` so it is all I need I think.

Reply:
@WoLewicki Sadly, I don't know if it's all you need. I see that this PR has been raised some time ago, that may be related to your error: https://github.com/software-mansion/react-native-screens/issues/2027
Can you try if applying the fix from pager-view fixes that issue as well?
I might also take a look on that fix, if you don't have any time to spare!